src/app/components/applications/SSV/MyAccount/components/Validator/FundingNewValidator/FundingNewValidator.tsx

```diff
@@ -30,13 +31,16 @@ const FundingNewValidator = () => {
   const OPTION_DEPOSIT_ADDITIONAL_FUNDS = 2;
   const processStore: ProcessStore = stores.Process;
   const clusterStore: ClusterStore = stores.Cluster;
+  const validatorStore: ValidatorStore = stores.Validator;
   const process: SingleCluster = processStore.getProcess;
   const [checkedId, setCheckedId] = useState(0);
   const [depositSSV, setDepositSSV] = useState<string | number>(0);
   const [errorMessage, setErrorMessage] = useState({ text:'', link: { text:'', path:'' } });
   const [showErrorMessage, setShowErrorMessage] = useState(false);
   const cluster = process.item;
-  const newBurnRate = clusterStore.getClusterNewBurnRate(cluster, cluster.validator_count + 1);
+  const currentValidatorsCount = cluster.validatorCount ? cluster.validatorCount : cluster.validator_count;
```

Comment:
Why do we use validatorCount and validator_count on the same object? I see it is most likely some kind of work around introduced previously. (in ValidatorFlow.tsx withdrawSsv function) but please change this

Reply:
V